src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
+import { Comment } from './Comment';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | null,
+  comments: Comment[] | null,
```

Comment:
Usually left empty array if don’t have comments

```suggestion
  comments: Comment[],
```